packages/helpers/src/polyfills/elementScroll.js

```diff
@@ -4,6 +4,8 @@
 // https://github.com/idmadj/element-scroll-polyfill/blob/master/index.js
 
 (function() {
+  if ((Element.prototype.scroll && Element.prototype.scroll) || !document) return;
```

Comment:
huh why is `Element.prototype.scroll` repeated two times? And why is it necessary here if there's a check on line  37?

Reply:
oh one of them should be `Element.prototype.scrollTo`, my copy paste error, and I guess there might be a case there is a `scroll` function but not `scrollTo`?